src/main/java/org/jabref/logic/citationkeypattern/BracketedPattern.java

```diff
@@ -1248,15 +1185,20 @@ private static String generateInstitutionKey(String content) {
             return "";
         }
 
+        Matcher matcher = INLINE_ABBREVIATION.matcher(content);
+        if (matcher.find()) {
+            return LatexToUnicodeAdapter.format(matcher.group());
+        }
+
         String result = content;
-        result = unifyDiacritics(result);
-        result = result.replaceAll("^\\{", "").replaceAll("}$", "");
-        Matcher matcher = ABBREVIATIONS.matcher(result);
-        if (matcher.matches()) {
-            return matcher.group(1);
+        try {
+            result = LatexToUnicodeAdapter.format(content);
+        } catch (IllegalArgumentException e) {
+            LOGGER.warn(content + " could not be converted to unicode. This can result in an incorrect or missing institute citation key");
         }
```

Comment:
This _should_ be caught earlier in the code but perhaps it is good to leave it in?